src/components/Class/ClassCard/index.js

```diff
@@ -183,60 +197,141 @@ function ClassCard({ item, editClass, enroll, style }) {
       dispatch(classActions.dropOutClass(Id));
     });
   };
+
+  const EnrolledAndTimer = () => {
+    const [Timer, setTimer] = useState(TimeLeft(item.start_time));
+    const ONE_MINUTE = 60000; //millisecs
+    setInterval(() => {
+      setTimer(TimeLeft(item.start_time));
+    }, ONE_MINUTE);
+    return (
+      <>
+        {Timer == "joinNow" ? (
+          <a
+            style={{
+              textDecoration: "none",
+            }}
+            href={item.meet_link}
+            target="_blank"
+          >
+            <Button variant="contained" fullWidth>
+              Join Now
+            </Button>
+          </a>
```

Comment:
Can use [`ExternalLink`](https://github.com/navgurukul/bhanwari-devi/blob/08eaeab3030736341d3b7a3685c9477e3bb40de6/src/components/common/ExternalLink/index.js) component here:
```suggestion
          <ExternalLink
            style={{
              textDecoration: "none",
            }}
            href={item.meet_link}
          >
            <Button variant="contained" fullWidth>
              Join Now
            </Button>
          </ExternalLink>
```